website/app/styles/doc-components/algolia-search/form.scss

```diff
@@ -50,13 +50,6 @@ $doc-algolia-search-modal-search-input-height: 44px;
 .aa-InputWrapperPrefix { left: 10px; }
 .aa-InputWrapperSuffix { right: 10px; }
 
-
-// this layer is useless for us
-
-.aa-Label {
```

Comment:
I don't remember why it was added, but I checked in a browser and it doesn't seem to be needed